group changes webhooks and pushes (#929)
livechat/livechat-public-docs: content/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx

Also, `misdirected_connection` returns the correct `region` value in the optional `data` object.
With this piece of information, the client is able to figure out where it should be connected.

## Configuration

### `group_created`

> Informs that a group has been created on the license.

Reviewer: ```suggestion
Informs that a group was within a license.
```

<CodeResponse title={'Sample push payload'}>

```json
{

>   "id": 42,

Reviewer: Incorrect indentation

	"name": "Sales",
	"language_code": "en",
	"agent_priorities": {
    "agent@example.com": "normal",
    "other_agent@example.com": "first"
  }
}
```

</CodeResponse>

#### Specifics

|                        |                                                                                 |
| ---------------------- | ------------------------------------------------------------------------------- |
| **Action**             | [`create_group`](/management/configuration-api/#create_group)                   |
| **Webhook equivalent** | [`group_created`](/management/configuration-api/#group_created)                 |

> ### `group_deleted`

Reviewer 2: Any reason the order is different in the RTM version?

Reviewer: Different than where?

Reviewer 2: The order of events in non-RTM was: created, updated, deleted, while in RTM it was: created, deleted, updated. Łukasz has now fixed that.

> Informs that a group has been deleted on the license.

Reviewer: ```suggestion
Informs that a group was deleted within a license.
```

<CodeResponse title={'Sample push payload'}>

```json
{
  "id": 42
}
```

</CodeResponse>

#### Specifics

|                        |                                                                                 |
| ---------------------- | ------------------------------------------------------------------------------- |
| **Action**             | [`delete_group`](/management/configuration-api/#delete_group)                   |
| **Webhook equivalent** | [`group_deleted`](/management/configuration-api/#group_deleted)                 |

### `group_updated`

> Informs that a group has been updated on the license.  Payload only includes fields that were changed.

Reviewer: ```suggestion
Informs that a group was updated within a license.  The push payload only includes the updated fields.
```
